bucket/prismlauncher-qt5.json

```diff
@@ -3,12 +3,43 @@
     "description": "An open source Minecraft launcher with the ability to manage multiple instances, accounts and mods. Focused on user freedom and free redistributability. Qt 5 build that still supports legacy Windows like 8.1 and 7.",
     "homepage": "https://prismlauncher.org/",
     "license": "GPL-3.0-only",
-    "notes": "To add Prism Launcher file association options for .ZIPs and .MRPACKS, run this: \"$dir\\install-associations.reg\"",
-    "url": "https://github.com/PrismLauncher/PrismLauncher/releases/download/7.1/PrismLauncher-Windows-MSVC-Legacy-7.1.zip",
-    "hash": "904efed4e15d0274f25f6fd2bdb9cb61dbbef2ad90dfc9761abb3612869df89f",
+    "notes": [
+        "To add Prism Launcher file association options for .ZIPs and .MRPACKS, run this: \"$dir\\install-associations.reg\"",
+        "This package is now using the portable version of Prism Launcher, and data should have been migrated automatically.",
+        "If you are using a global install, you will need to copy your data from AppData to the new Scoop PrismLauncher persist directory"
+    ],
+    "url": "https://github.com/PrismLauncher/PrismLauncher/releases/download/7.1/PrismLauncher-Windows-MSVC-Legacy-Portable-7.1.zip",
+    "hash": "4c8217ad3f37c5fe1df52cc7b1018f73a4cb59aea4a4b7c7924f3398c1de2745",
     "suggest": {
         "Microsoft Visual C++ Redistributables": "extras/vcredist2022"
     },
+    "pre_install": [
+        "$appdataPath = \"$HOME\\AppData\\Roaming\\PrismLauncher\"",
+        "if ( (Test-Path -Path $appdataPath\\*) -and (!(Test-Path -Path $persist_dir\\*)) -and (!($global)) ) {",
+        "    Write-Warning \"Migrating data from $appdataPath to $persist_dir (this may take a while)\"",
+        "    New-Item -Type Directory -Path $persist_dir | Out-Null",
+        "    Copy-Item -Recurse -Force $appdataPath\\* $persist_dir\\",
+        "}",
+        "",
+        "$logDir = \"$original_dir\\logs\"",
+        "if ((!(Test-Path -Path $logDir))) {",
+        "    New-Item -Type Directory -Path $logDir | Out-Null",
+        "}",
+        "",
+        "$acl = Get-Acl $logDir",
```

Comment:
Setting permissions on the logs folder shouldn't be needed for global installs. To my knowledge, the Scoop `persist` folder is already writable to users, otherwise things like making instances, accounts, etc... wouldn't work as well.

Reply:
massive brain fart...

Reply:
Lol, it's fair enough, I was being confusing about it to be fair.